change/@azure-msal-browser-59aee50c-dfa5-47b3-aba3-935707c03d9f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "ATS- event vis change",
```

Comment:
Please remove the abbreviations here and throughout the code for readability.
`Adding page visibility change event for silent calls like AcquireTokenSilent`

Reply:
fixed